ci/docker/yum/Dockerfile

```diff
@@ -52,11 +52,11 @@ RUN yum -y update; \
     # Devel packages that ATS needs
     yum -y install openssl-devel tcl-devel expat-devel pcre-devel libcap-devel hwloc-devel libunwind-devel \
     xz-devel libcurl-devel ncurses-devel jemalloc-devel GeoIP-devel kyotocabinet-devel luajit-devel \
-    perl-ExtUtils-MakeMaker perl-Digest-SHA; \
+    brotli-devel ImageMagick-devel ImageMagick-c++-devel perl-ExtUtils-MakeMaker perl-Digest-SHA; \
```

Comment:
What would be a criteria for adding packages here? At least mysql-devel (or mariadb-devel) is not here.

Also, the comment above says "Devel packages that *ATS* needs". It might be better to split yum command into one for ATS itself and one for plugins to clarify which packages are actually for ATS.

Reply:
Yeh, it's mostly arbitrary to figure out what packages we need. For MariaDB, I'm ok if we want to add that, however, I feel that plugin is not useful, and we likely should nuke it (because it's using a blocking API, stalling all net-threads).

And sure, we can split (but that makes it another checkpoint / UUID commit, because of how Docker works :). Not a big deal though, but I tried to merge them as much as possible to make it less slow.

Reply:
OK, then I don't think we need to add mariadb-devel. I thought it's not so fair to exclude some plugins without reasons. I didn't check carefully, but seems like other plugins are already covered.

If we want to avoid multiple yum commands we can insert comments between package names like below:
```sh
yum -y install pkg-abc \
`# This is a comment` \
pkg-xyz
```

Reply:
Yeh, it already does that :) Feel free to make a second PR that cleans things up.

Reply:
Does it? I may be missing the places but many of existing comments are between commands.

Reply:
Yeh, but it's done the way you describe in a few places.

```
    # Compilers
    yum -y install ccache make pkgconfig bison flex gcc-c++ clang \
    # Autoconf
    autoconf automake libtool \
    # Various other tools
    git rpm-build distcc-server file; \
```

Reply:
Ah, I was confused because those look like command lines :)